backend/utils/handleLog.js

```diff
@@ -395,7 +396,8 @@ async function _processEventListingCreated({ event }) {
   const shopConfig = JSON.parse(raw.toString())
 
   // Update the config.json listingId field and write it back to disk.
-  shopConfig.listingId = listingId
+  const netPath = `networks[${shop.networkId}]`
+  set(shopConfig, `${netPath}.listingId`, listingId)
```

Comment:
ah good catch! Thanks for fixing this. I did not realize the listingId was stored in the networks object... my bad. I'll adjust the unit tests accordingly.